go/protocol/chat1/api.go

```diff
@@ -390,11 +390,11 @@ type ConvSummary struct {
 	ActiveAt     int64                     `codec:"activeAt" json:"active_at"`
 	ActiveAtMs   int64                     `codec:"activeAtMs" json:"active_at_ms"`
 	MemberStatus string                    `codec:"memberStatus" json:"member_status"`
-	ResetUsers   []string                  `codec:"resetUsers" json:"reset_users"`
+	ResetUsers   []string                  `codec:"resetUsers,omitempty" json:"reset_users,omitempty"`
```

Comment:
It looks like a couple optional keys were already defined by me in the AVDL. I have a separate PR up to finish that off.